lua/refactoring/init.lua

```diff
@@ -2,12 +2,14 @@ local refactors = require("refactoring.refactor")
 local Config = require("refactoring.config")
 
 local M = {}
+local config = {}
 
 function M.setup(config)
-    Config.setup(config)
+    config = Config.setup(config)
```

Comment:
we don't use the `config` parameter within the config.setup, but that is fine, but that is fine.

Reply:
I know, but it's marked as a TODO 🤷‍♀️